contributions/api/utils/mongoutils.py

```diff
@@ -19,21 +19,29 @@
 from bson import ObjectId
 from flask import make_response, json
 from bson.json_util import dumps
-from pymongo import MongoClient, ASCENDING
+from pymongo import MongoClient, ASCENDING, TEXT
 from models.contribution import Contribution
 from utils import query_params
 from utils import jsonutils
 from utils import adminutils
 
 client_contribution = MongoClient(cfg.MONGO_CONTRIBUTION_URL, connect=False)
 db_contribution = client_contribution[cfg.CONTRIBUTION_DB_NAME]
-coll_contribution = db_contribution[cfg.CONTRIBUTION_COLL_NAME]
-coll_contribution.create_index([("name", ASCENDING)], background=True)
-coll_contribution.create_index([("capabilities.name", ASCENDING)], background=True)
-coll_contribution.create_index([("talents.name", ASCENDING)], background=True)
-
-coll_reviewer = db_contribution[cfg.REVIEWER_COLL_NAME]
-coll_reviewer.create_index([("name", ASCENDING)], background=True)
+coll_contribution = db_contribution[cfg.CONTRIBUTION_COLL_NAME]  # set contribution collection
+# drop old indexes in contribution collection
+coll_contribution_indexes = coll_contribution.index_information().keys()
+# get all indexes except id and text index
+coll_contribution_old_indexes = [index for index in coll_contribution_indexes if index not in ['_id_', 'text_index']]
+for index_name in coll_contribution_old_indexes:
+    coll_contribution.drop_index(index_name)
```

Comment:
It may be better to drop specific indexes (rather than all other indexes). This piece of code could create issues when we add new indexes. What do you think? If you agree, please create a follow-up issue to fix this. Thanks.

Reply:
@minump, I Just wanted to ensure that you saw my message above. Thanks.